development/language/guidelines.rst

```diff
@@ -0,0 +1,190 @@
+                          topic_notify.txt
+                       admin_activate.txt
+                       admin_send_email.txt
+                       admin_welcome_activated.txt
+                       admin_welcome_inactive.txt
+                       bookmark.txt
+                       contact_admin.txt
+                       coppa_resend_inactive.txt
+                       coppa_welcome_inactive.txt
+                       email_notify.txt
+                       forum_notify.txt
+                       group_added.txt
+                       group_request.txt
+                       index.htm (optional)
+                       installed.txt
+                       newtopic_notify.txt
+                       pm_report_closed.txt
+                       pm_report_deleted.txt
+                       post_approved.txt
+                       post_disapproved.txt
+                       post_in_queue.txt
+                       privmsg_notify.txt
+                       profile_send_email.txt
+                       profile_send_im.txt
+                       quote.txt
+                       report_closed.txt
+                       report_deleted.txt
+                       report_pm.txt
+                       report_post.txt
+                       topic_approved.txt
+                       topic_disapproved.txt
+                       topic_in_queue.txt
+                       topic_notify.txt
+                       user_activate.txt
+                       user_activate_inactive.txt
+                       user_activate_passwd.txt
+                       user_reactivate_account.txt
+                       user_remind_inactive.txt
+                       user_resend_inactive.txt
+                       user_welcome.txt
+                       user_welcome_inactive.txt
+                    app.php
+                    AUTHORS (optional)
+                    captcha_qa.php
+                    captcha_recaptcha.php
+                    cli.php
+                    CHANGELOG (optional)
+                    common.php
+                    groups.php
+                    help_bbcode.php
+                    help_faq.php
+                    index.htm (optional)
+                    install.php
+                    iso.txt (
+                    LICENSE
+                    mcp.php
+                    memberlist.php
+                    migrator.php
+                    plupload.php
+                    posting.php
+                    README (optional)
+                    search.php
+                    ucp.php
+                    VERSION (optional)
+                    viewforum.php
+                    viewtopic.php
+              styles/
+                 prosilver/
+                    theme/
+                       {iso}/
+                          icon_user_online.gif
+                          index.htm (optional)
+                          stylesheet.css
+                 subsilver2/
```

Comment:
This will be removed for the 3.2 documentation?

Reply:
Yes, merge upstream and then I'll make a additional pr for the changes for 3.2.